### ISSUE TITLE
Exclude complex `transition` for `declaration-block-no-redundant-longhand-properties`

### ISSUE THREAD

**Issue by AlexWayfer** [stylelint/stylelint#4074]
> What is the problem you're trying to solve?

I want to write complex transitions with multiple properties.

> What solution would you like to see?

I think, this is valid case for `declaration-block-no-redundant-longhand-properties`:

```css
transition-property: padding;
transition-duration: 400ms;
transition-timing-function: ease-in-out;
transition-delay: 0s;
```

Because the shorthand variant is OK:

```css
transition: padding 400ms ease-in-out 0s;
```

But this should not be offensed by Stylelint:

```css
transition-property: padding, margin, border;
transition-duration: 400ms; // or even `400ms, 400ms, 0s`
transition-timing-function: ease-in-out;
transition-delay: 0s, 0s, 400ms;
```

Because the shorthand variant is ugly:

```css
transition: padding 400ms ease-in-out 0s, margin 400ms ease-in-out 0s, border 400ms ease-in-out 400ms;
```

**Comment by evilebottnawi** [comment]
Just disable rule, or use `/* stylelint-disable-next-line */`

**Comment by evilebottnawi** [comment]
For other developers it can be not ugly

**Comment by AlexWayfer** [comment]
> Just disable rule

Yes, I've done this.

> For other developers it can be not ugly

OK, you can close this issue, if you think, that this is OK and work-around like inline disabling is OK too.

**Comment by evilebottnawi** [comment]
You can enable other rules and use this style:
```css
transition: padding 400ms ease-in-out 0s, 
  margin 400ms ease-in-out 0s, 
  border 400ms ease-in-out 400ms;
```

or this:
```css
transition: 
  padding 400ms ease-in-out 0s, 
  margin 400ms ease-in-out 0s, 
  border 400ms ease-in-out 400ms;
```

As you prefer :+1:

**Comment by AlexWayfer** [comment]
OK, thanks. I'll close it for now.
